.github/workflows/build-gems.yml

```diff
@@ -94,7 +98,12 @@ jobs:
             rubyPlatform: x86_64-darwin
           - os: macos-latest
             rubyPlatform: arm64-darwin
+          - os: ubuntu-latest
+            rubyVersion: "3.3"
```

Comment:
That's saying you will no longer run this for Ruby 3.3 on non-musl. Is this really what you meant?

Reply:
No, this actually adds a _new_ single entry to the matrix due to the field below this one for setting a container different than an existing metric. See https://github.com/temporalio/sdk-ruby/actions/runs/13185031590. In fact, since we install Ruby a different way for Alpine, technically I could set this to any value because it is not used for this specific matrix entry.

Reply:
> due to the field below this one for setting a container different than an existing metric

Oh, right… very subtle. I'd suggest a comment.